website/docs/how-to/using-a-capacitor-plugin.md

```diff
@@ -2,3 +2,276 @@
+
+### CapacitorCamera
+
+The Camera API provides the ability to take a photo with the camera or choose an existing one from the photo album.
+
+### CapacitorClipboard
+
+The Clipboard API enables copy and pasting to/from the system clipboard.
+
+### CapacitorDevice
+
+The Device API exposes internal information about the device, such as the model and operating system version, along with user information such as unique ids.
+
+### CapacitorDialog
+
+The Dialog API provides methods for triggering native dialog windows for alerts, confirmations, and input prompts
+
+### CapacitorFilesystem
+
+The Filesystem API provides a NodeJS-like API for working with files on the device.
+
+### CapacitorGeolocation
+
+The Geolocation API provides simple methods for getting and tracking the current position of the device using GPS, along with altitude, heading, and speed information if available.
+
+### CapacitorHaptics
+
+The Haptics API provides physical feedback to the user through touch or vibration.
+
+### CapacitorKeyboard
+
+The Keyboard API provides keyboard display and visibility control, along with event tracking when the keyboard shows and hides.
+
+### CapacitorLocalNotifications
+
+The Local Notifications API provides a way to schedule device notifications locally (i.e. without a server sending push notifications).
+
+### CapacitorNetwork
+
+The Network API provides network and connectivity information.
+
+### CapacitorPushNotifications
+
+The Push Notifications API provides access to native push notifications.
+
+### CapacitorScreenReader
+
+The Screen Reader API provides access to TalkBack/VoiceOver/etc. and provides simple text-to-speech capabilities for visual accessibility.
+### CapacitorShare
+
+The Share API provides methods for sharing content in any sharing-enabled apps the user may have installed.
+
+### CapacitorSplashScreen
+
+The Splash Screen API provides methods for showing or hiding a Splash image.
+
+### CapacitorStatusBar
+
+The StatusBar API Provides methods for configuring the style of the Status Bar, along with showing or hiding it.
+
+### CapacitorStorage
+
+The Storage API provides a simple key/value persistent store for lightweight data.
+
+### CapacitorTextZoom
+
+The Text Zoom API provides the ability to change Web View text size for visual accessibility.
+
+### CapacitorToast
+
+The Toast API provides a notification pop up for displaying important information to a user. Just like real toast!
+
+</TabItem>
+<TabItem value="android">
+
+## Usage 
+In order to use a Capacitor Plugin, you need to install the plugin as a dependency in your `build.gradle` file.
+
+```groovy
+dependencies {
+    implementation 'io.ionic:portals:0.2.0'
+    implementation 'com.capacitorjs.storage:1.2.0'
+}
+```
+
+:::warning
+Make sure that the versions in your `Podfile`, `build.gradle`, and `package.json` all match! Otherwise you will run into errors.
+:::
```

Comment:
Suggesting making the Admonition less scary/red and wording this piece for Android specifically in the Android tab.

```suggestion
:::caution
Make sure that the versions in your `build.gradle` and `package.json` match! Otherwise you could run into errors.
:::
```